tap-snapshots/test-unit-ts.mjs-TAP.test.js

```diff
@@ -0,0 +1,34 @@
+/* IMPORTANT
+ * This snapshot file is auto-generated, but designed for humans.
+ * It should be checked into source control and tracked carefully.
+ * Re-generate by setting TAP_SNAPSHOT=1 and running tests.
+ * Make sure to inspect the output below.  Do not ignore changes!
+ */
+'use strict'
+exports[`test/unit/ts.mjs TAP > Pre-Processor processes input correctly 1`] = `
+Object {
+  "extension": ".mjs",
+  "source": "export const dummy = 42;\\nexport const dummy2 = [42, 42, 42];\\n",
+}
+`
+
+exports[`test/unit/ts.mjs TAP > Pre-Processor processes input correctly 2`] = `
+Object {
+  "extension": ".mjs",
+  "source": "import { dummy } from \\"/usr/local/google/home/jlondono/Documents/presm/test/fixtures/tsmodule1.ts\\";\\nconsole.log(dummy);\\n",
```

Comment:
I assume this would fail on CI?

Reply:
Oops, yup, this should be changed to account for resolving specifiers to relative paths, not absolute.   This feature is coming to the ts-loader soon.

Reply:
Fixed this, it now fails for the known cause of replacing specifiers with absolute paths instead of relative one 👍